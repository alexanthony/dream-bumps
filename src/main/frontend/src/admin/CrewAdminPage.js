// @flow
import React, { Component, PropTypes } from 'react';
import { connect } from 'react-redux';

/**
 * UserAdminPage
 */
const CrewAdminPage = props => (
  <div>
    {props.children}
  </div>
);

CrewAdminPage.propTypes = {
  children: PropTypes.node
};

export { CrewAdminPage };

export default CrewAdminPage;
