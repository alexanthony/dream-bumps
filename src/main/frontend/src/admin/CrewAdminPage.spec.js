import React from 'react';
import { shallow } from 'enzyme';
import { CrewAdminPage } from './CrewAdminPage';
import toJson from 'enzyme-to-json';

test('Renders properly', () => {
  const fetchCrews = jest.fn();
  const component = shallow(
    <CrewAdminPage
      fetchCrews={fetchCrews}
    >
      <div className="random div" />
    </CrewAdminPage>
  );
  expect(component).toBeDefined();
  expect(toJson(component)).toMatchSnapshot()
});
