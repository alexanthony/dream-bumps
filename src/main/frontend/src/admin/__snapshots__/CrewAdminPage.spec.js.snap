exports[`test Renders properly 1`] = `
<div>
  <div
    className="random div" />
</div>
`;
